src/psbt.h

```diff
@@ -713,7 +713,7 @@ struct PSBTOutput
     CScript witness_script;
     std::map<CPubKey, KeyOriginInfo> hd_keypaths;
     XOnlyPubKey m_tap_internal_key;
-    std::optional<TaprootBuilder> m_tap_tree;
+    std::optional<std::vector<std::tuple<uint8_t, uint8_t, CScript>>> m_tap_tree;
```

Comment:
nit: What's the reason for keeping it optional now it's a vector?

Reply:
Dropped the optional